modules/ROOT/pages/security/authorization.adoc

```diff
@@ -115,6 +118,9 @@ type Post @authorization(validate: [
 }
 ----
 
+[NOTE]
+If there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.
```

Comment:
```suggestion
====
In case there is no `operations` argument with a list of operations, the GraphQL Library treats the authorization configuration as if the full list of operations had been provided.
====
```